source/PlayerInfo.cpp

```diff
@@ -1356,7 +1366,27 @@ void PlayerInfo::SetShipOrder(const vector<shared_ptr<Ship>> &newOrder)
 	// Check if the incoming vector contains the same elements
 	if(std::is_permutation(ships.begin(), ships.end(), newOrder.begin()))
 	{
+		auto OldFirstShip = ships[0];
```

Comment:
according to our style, variable names should start with a lowercase letter
```suggestion
		auto oldFirstShip = ships[0];
```